modules/api-gateway/src/iam-policy.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
the user should have only access to our new created api gateway,
It seems this policy allows access to all api gateways on the account